src/main/java/marquez/api/resources/DatasourceResource.java

```diff
@@ -61,6 +64,25 @@ public Response list(
     return Response.ok(DatasourceResponseMapper.toDatasourcesResponse(datasources)).build();
   }
 
+  @GET
+  @ResponseMetered
+  @ExceptionMetered
+  @Timed
+  @Path("/{datasourceId}")
+  @Produces(APPLICATION_JSON)
+  public Response get(@PathParam("datasourceId") final DatasourceId datasourceId)
+      throws MarquezServiceException {
+
+    final Optional<Datasource> datasource;
+    datasource = datasourceService.get(datasourceId.getValue());
+
+    if (datasource.isPresent()) {
+      return Response.ok(DatasourceResponseMapper.map(datasource.get())).build();
+    } else {
+      return Response.status(Response.Status.NOT_FOUND).entity("Datasource not found").build();
+    }
+  }
+
   @POST
   @ResponseMetered
   @ExceptionMetered
```

Comment:
For readability, and consistency across our resource classes (something we are working towards), please define the method ordering as:

```java
class DatasourceResource {
  create()
  get()
  list()
}

Reply:
Fixed in an update to #394.